automatix/__init__.py

```diff
@@ -192,9 +38,12 @@ def main():
             batch_items = list(DictReader(csvfile))
         LOG.notice('Detected batch processing from CSV file.')
 
+    i = 0
     for row in batch_items:
```

Comment:
Hier könnte man auch schön über die Liste enumerieren :wink: 
`for i, row in enumerate(batch_items):`

Reply:
Ich wusste doch, dass das auch einfacher geht :-)